modules/lang-painless/spi/src/main/java/org/elasticsearch/painless/spi/WhitelistLoader.java

```diff
@@ -502,20 +502,20 @@ public static Whitelist loadFromResourceFiles(Class<?> resource, Map<String, Whi
                         if (argumentKeyValue.length != 2) {
                             throw new IllegalArgumentException("invalid annotation: expected key=\"value\" [" + line + "]");
                         }
-                        
+
```

Comment:
After we get this into 7.x, let's fix these up in master to cleanup the backport.

Reply:
Agreed, we should just do an entire whitespace PR.